how-to-guides/data-versioning/notebooks/Organize_and_share_dataset_versions.ipynb

```diff
@@ -407,27 +521,35 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "Now train a model and log the test score to Neptune."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
    "outputs": [],
    "source": [
     "from sklearn.ensemble import RandomForestClassifier\n",
     "\n",
     "TEST_DATASET_PATH = \"../datasets/tables/test.csv\"\n",
     "\n",
-    "PARAMS = {\n",
+    "params = {\n",
```

Comment:
Same comment as for the earlier file: curious why this change here?

Reply:
these are the ones we are logging and can change between runs